db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
```

Comment:
Provide `related_name` for ForeignKey fields

Reply:
![зображення_2024-04-04_181926818](https://github.com/mate-academy/py-game-models/assets/64077126/26654d47-b1c9-4fad-8aee-9ff6c8be93c8)
i don't change this line, because its call Atribute erros in pytest side